scripts/analysis/amwg_table.py

```diff
@@ -297,5 +303,44 @@ def _write_html(f, out):
         hfil.write(html)
         hfil.write(ending)
 
+def _df_comp_table(write_html,output_location,case_names):
+    import pandas as pd
+
+    # * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+    # first csv is case and second is baseline/obs 
+    # * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+    output_csv_file_comp = output_location / "amwg_table_comp.csv"
+    case = output_location/f"amwg_table_{case_names[0]}.csv"
+    baseline = output_location/f"amwg_table_{case_names[1]}.csv"
+    
+    df_case = pd.read_csv(case)
+    df_base = pd.read_csv(baseline)
+    df_comp = pd.DataFrame(dtype=object)
+
+    df_comp[['variable','unit','case']] = df_case[['variable','unit','mean']]
+    df_comp['baseline'] = df_base[['mean']]
+    
+    # ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * !
+    # Should this be an absolute value difference, or case minus baseline?
+    # ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * ! * !
+    df_comp['diff'] = df_comp['baseline'].values-df_comp['case'].values
```

Comment:
Should this difference be flipped?  That way positive numbers would mean that the test case has a positive bias, as opposed to meaning that the test case has a negative bias (which might require users to do a sign-flip in their head).